.github/workflows/build-and-deploy.yml

```diff
@@ -8,10 +8,10 @@ on:
         type: choice
         default: development
         options:
-        - review
-        - development
-        - test
-        - production
+          - review
```

Comment:
should have concurrency at the workflow level e.g. https://github.com/DFE-Digital/publish-teacher-training/blob/main/.github/workflows/build-and-deploy.yml#L2